clifford/_layout.py

```diff
@@ -568,6 +619,13 @@ def comp_func(Xval):
             return Yval
         return comp_func
 
+    def gradeInvol(self, mv):
+        return grade_involution_func(mv, self._basis_blade_order.grades)
```

Comment:
This is a bit weird - why not make this `grade_invol_func` to match the others, and precompute `np.power(-1, grades_array)` like we do for the adjoint function?

Reply:
yeah, to be honest that is a better way to go, I did originally have it that way so not entirely sure how I ended up with this setup